docusaurus.config.js

```diff
@@ -85,6 +85,15 @@ module.exports = {
             from: '/guides/key-manager/execute-relay-call',
             to: '/guides/key-manager/execute-relay-transactions',
           },
+          // Smart Contracts new structure
+          {
+            from: '/standards/smart-contracts/introduction',
+            to: '/contracts/introduction',
+          },
+          {
+            to: '/contracts/interface-ids',
+            from: '/standards/smart-contracts/interface-ids',
```

Comment:
```suggestion
            from: '/standards/smart-contracts/interface-ids',
            to: '/contracts/interface-ids',
```

just for consistency, makes tihngs easier to read as they are all in the same order